python/hsfs/core/feature_monitoring_config_engine.py

```diff
@@ -69,17 +115,34 @@ def enable_feature_monitoring_config(
         alert_config: str,
         scheduler_config: str,
         description: Optional[str] = None,
-        feature_group_id: Optional[int] = None,
-        feature_view_id: Optional[int] = None,
-        feature_view_name: Optional[str] = None,
-        feature_view_version: Optional[str] = None,
     ) -> FeatureMonitoringConfig:
+        """Enable feature monitoring for a feature.
+
+        Args:
+            feature_name: str, required
+                Name of the feature to monitor.
+            name: str, required
+                Name of the monitoring configuration.
+            detection_window_config: Dict[str, Any], required
+                Configuration of the detection window.
+            reference_window_config: Dict[str, Any], required
+                Configuration of the reference window.
+            statistics_comparison_config: Dict[str, Any], required
+                Configuration of the statistics comparison.
+            alert_config: str, required
```

Comment:
alerts should be optional
```suggestion
            alert_config: str, optional
```